packages/webapp/src/components/CardWithStatus/ManagementPlanCard/styles.module.scss

```diff
@@ -8,7 +9,7 @@
   font-size: 16px;
   line-height: 24px;
   font-style: normal;
-  font-family: 'Open Sans', 'SansSerif', serif;
+  @include fontFamily();
```

Comment:
Maybe a silly question but is it possible or a bad idea to set this globally and remove font family from all components?

Reply:
100%! But it seems for some components we need to specify it at the element level because we need to overwrite existing MUI styles -- it's probably only a handful of cases and not all of the different instances that we have, but I figured I'd figure that out later 😅 but you're totally right, we shouldn't be specifying this a hundred times

Reply:
Oh yeah I forgot about MUI!